code/modules/mob/living/carbon/human/species_types/dwarf.dm

```diff
@@ -0,0 +1,43 @@
+/datum/species/dwarf //not to be confused with the genetic manlets
+	name = "Dwarf"
+	id = "dwarf"
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS,NOBREATH)
+	mutant_bodyparts = list("tail_human", "ears", "wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	limbs_id = "human"
+	use_skintones = 1
+	speedmod = 1
+	skinned_type = /obj/item/stack/sheet/animalhide/human
+	brutemod = 0.9
+	coldmod = 0.85
+	punchdamagehigh = 11
```

Comment:
Dwarves seem like tough little fuckers, so it seemed appropriate that they hit harder.